README.md

```diff
@@ -20,6 +20,7 @@ Contributing to [Apache Maven Resources Plugin](https://maven.apache.org/plugins
 [![ASF Jira](https://img.shields.io/endpoint?url=https%3A%2F%2Fmaven.apache.org%2Fbadges%2Fasf_jira-MRESOURCES.json)][jira]
 [![Apache License, Version 2.0, January 2004](https://img.shields.io/github/license/apache/maven.svg?label=License)][license]
 [![Maven Central](https://img.shields.io/maven-central/v/org.apache.maven.plugins/maven-resources-plugin.svg?label=Maven%20Central)](https://search.maven.org/artifact/org.apache.maven.plugins/maven-resources-plugin)
+[![Maven Central](https://img.shields.io/maven-central/v/org.apache.maven.plugins/maven-resources-plugin.svg?label=Maven%20Central&versionPrefix=3.)](https://search.maven.org/artifact/org.apache.maven.plugins/maven-resources-plugin)
```

Comment:
What are Apache policies on embedding third party content and services in a page like this? I know we're not supposed to be using third party JavaScript.

Reply:
Good questions ... but we use a badges at GitHub in all our projects ....

Reply:
I checked sources of generated GitHub page ... and external images are translated into something:

```
<img src="https://camo.githubusercontent.com/79cd61....
```
so should be ok - there are no third party content